src/core_plugins/kibana/public/home/components/tutorial/tutorial.js

```diff
@@ -313,10 +313,15 @@ export class Tutorial extends React.Component {
     }
     return (
       <EuiPage className="home">
+        <EuiPageBody>
 
-        <EuiLink href="#/home">Home</EuiLink> / <EuiLink href="#/home/tutorial_directory">Add Data</EuiLink>
-        <EuiSpacer size="s" />
-        {content}
+          <div>
```

Comment:
Why is now wrapped in a `div`?

Reply:
@cchaos can you answer this? It's part of the code you made that I merged into this PR.